charmcraft/commands/store/store.py

```diff
@@ -210,3 +210,26 @@ def create_library_revision(self, charm_name, lib_id, api, patch, content, conte
         response = self._client.post(endpoint, payload)
         result = _build_library(response)
         return result
+
+    def get_libraries_tips(self, libraries):
+        """Get the tip details for several libraries at once."""
```

Comment:
This could use a bit more description on what it's doing, input, outputs, etc. It took me some time to parse what's happening.  

Seems like this queries a set of libraries referenced by ID, or charm name [+ library name]. It would help if these inputs could be encapsulated into better defined parameters / class(es) such as `Library`

Reply:
Improved the docstring to describe better what's going on.

Regarding interfaces, it's better to keep each item as a dictionary, because if the key is present we need to send that info in the payload (otherwise we would need a specific token to flag that the key should not be sent). That said, we can improve this in the future if needed; #70 is related.